junebug/tests/helpers.py

```diff
@@ -284,6 +288,18 @@ def create_channel_from_id(self, redis, config, id, service):
         channel = yield Channel.from_id(redis, config, id, service)
         returnValue(channel)
 
+    @inlineCallbacks
+    def create_logging_router(self, service, config={}):
```

Comment:
We're not really creating a router specific to logging here, this is just a general test router. So maybe a better name for this function is `create_test_router`?